app/src/main/java/ca/klapstein/baudit/managers/BauditRemoteManager.java

```diff
@@ -5,4 +5,9 @@
  */
 public class BauditRemoteManager {
     private static final String TAG = "BauditRemoteManager";
+
+    public boolean uniqueID(String s){
```

Comment:
I would rename the input variable to something more descriptive.

I.E. is this supposed to intake a username string? if so why not name it `username` over `s`? This is 2018 we don't have to worry about bytes anymore make names easy to understand, **medium in length**, and descriptive.